theme_common/static/src/scss/mixins.scss

```diff
@@ -68,7 +68,7 @@
         }
         .blockquote-text {
             overflow: hidden;
-            font-size: $font-size-base;
```

Comment:
@Guillaume-gdi Let's not touch the theme_common/mixins.scss file. Those are old mixins only called in old snippets. If they still make sense, they can stay non responsive to further encourage not using this anymore.